### ISSUE TITLE
As a Program Chair, I would like that the system detect sessions that receive two strong rejections and remove them from the list of sessions to be reviewed so that I don't waste the precious time of reviewers with a session that we know by history that won't be selected

### ISSUE THREAD

**Issue by gmotta** [agile-alliance-brazil/submissions#274]
As a Program Chair, I would like that the system detects sessions that receive two strong rejections and remove them from the list of sessions to be reviewed from other Reviewers so that I don't waste the precious time of reviewers with a session that we know that won't be selected.

**Comment by hugocorbucci** [comment]
Updating this with a newer description:
Como coordenador de programa,
Quero que qualquer proposta que receba uma segunda rejeição de qualquer tipo seja considerada como completamente avaliada e não esteja mais disponível para novas avaliações nem entre na lista de sessões pendente avaliação

TL;DR: any two rejections. Only those two rejections will be shown to both track leads and authors.

**Comment by ehernandez** [comment]
up
